pom.xml

```diff
@@ -176,6 +176,12 @@
         <jodah-failsafe.version>0.9.5</jodah-failsafe.version>
         <apache.shiro.version>1.3.2</apache.shiro.version>
         <spark.version>2.5.5</spark.version>
+
+        <!-- gitflow-incremental-builder -->
+        <!-- See https://github.com/vackosar/gitflow-incremental-builder for a list of all the options -->
+        <gib.referenceBranch>refs/remotes/origin/master</gib.referenceBranch>
```

Comment:
Yes. I used master as the reference branch by default so that most of the time people don't have to worry about it. But you can change it if you use `-Dgib.referenceBranch=...`, which would allow us to use this for release branches as well.